aexpect/ops_linux.py

```diff
@@ -432,6 +432,29 @@ def grep(session, expr, path, check=False, flags="-a"):
     return output
 
 
+def grep_pipe(session, cmd, pattern, flags = "", check = False):
+    """
+    Invoke ``grep`` remotely searching for an expression in a command output.
+
+    :param session: vm session
+    :type session: ShellSession
+    :param str cmd: command that its output will be grepped
```

Comment:
I'm not native but this sounds strange. How about `command whose output will be grepped`?

Reply:
Much better, I'm not native either you can see that.